autoload/python_copy_reference.vim

```diff
@@ -62,7 +62,7 @@ endfunction
 
 function! python_copy_reference#_relative_path(reference)
   if !has_key(g:python_copy_reference, 'paths')
-    return reference
+    return a:reference
```

Comment:
Good catch, and this will be only triggered if the `paths` key is not set, that's why didn't encounter issue also in my local.

And could we rename `paths` here to `remove_prefixes` as discussed here?
https://github.com/ranelpadon/python-copy-reference.vim/pull/1#issuecomment-1369414402

Reply:
Sure, I've added another commit to change it, and one more to mention it in the README.

Reply:
Great, looks much better now. Thanks a lot! :)